src/db/db.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-console */
+
+import mongoose from 'mongoose';
+
+import config from '../config';
+
+export default function () {
+  mongoose.connect(config.mongodb.url, {
+    useMongoClient: true
+  });
+
+  mongoose.connection.on('connected', () => {
+    console.log(`Mongoose: connected to ${config.mongodb.url}`);
+  });
+
+  mongoose.connection.on('error', (err) => {
+    console.log(`Mongoose: connection error: ${err}`);
+  });
+
+  mongoose.connection.on('disconnected', () => {
+    console.log('Mongoose: connection disconnected');
+  });
+
+  return mongoose;
```

Comment:
i think better rename this file to `mongoose`, cause you export from it exactly `mongoose` object, but you can argue